patches/net/minecraft/world/inventory/Slot.java.patch

```diff
@@ -25,15 +34,6 @@
 +        return slot;
 +    }
 +
-+    /**
-+     * Checks if the other slot is in the same inventory, by comparing the inventory reference.
-+     * @param other
-+     * @return true if the other slot is in the same inventory
-+     */
-+    public boolean isSameInventory(Slot other) {
```

Comment:
I think moving this now would be a breaking change for binary compatibility (but not source compatibility), as while the JVM will search superclasses for methods with the same signature, I'm not sure it will search interfaces unless the calling code used `INVOKEINTERFACE`.